app/server/templates/email-link-action/text.hbs

```diff
@@ -1,4 +1,4 @@
-Hi! Thanks for signing up for HackIIITV 2019. To verify your email, follow this link: {{verifyUrl}}
+Hi! Thanks for signing up for HackIIITV 2020. To verify your email, follow this link: {{verifyUrl}}
 
 Thanks,
 HackIIITV Team
```

Comment:
Newline preferred.

Reply:
Newline inserted in the latest commit.
changes done as requested please review @aashutoshrathi sir.